### PR TITLE
Implement fuzzer weight setting.

### PR DESCRIPTION
**Pull request by letitz**
This allows manipulating the weight of individual FuzzerJob entries in the database.

Complements the `fuzz-target set` command.

### CODE REVIEW

**Review by alhijazi — APPROVED** [review]
LGTM, thanks!

**Comment by letitz** [comment]
A test seems broken at HEAD. @jonathanmetzman for review and to merge this once the problem is fixed (at which point, hopefully, tests will pass on this PR).

**Comment by jonathanmetzman** [comment]
Problem should be fixed

File: src/local/butler/weights.py [hunk 1]
```diff
@@ -316,6 +316,45 @@ def _aggregate_fuzzer_jobs(
...
+    print('No matching FuzzerJob entries found for ' +
+          f'fuzzer {fuzzer} and job {job}')
+    return
+
+  if len(fuzzer_jobs) > 1:
+    print('Bailing out! Multiple FuzzerJob entries found for ' +
+          f'fuzzer {fuzzer} and job {job}: {fuzzer_jobs}')
+    return
+
+  fj = fuzzer_jobs[0]
```
Comment (jonathanmetzman) [inline src/local/butler/weights.py:339]: Nit: I'm not against obvious abbreviations like this, but it's never done in CF ;-)
Can you call it fuzzer_job please?

Comment (letitz) [reply src/local/butler/weights.py]: Done! Also did it for 2 other places in the file.

**Comment by letitz** [comment]
PTAL when you have a minute! No urgency on this.

**Review by jonathanmetzman — APPROVED** [review]
lgtm

**Pull request merged**